cli/src/commandQuery.js

```diff
@@ -0,0 +1,30 @@
+const { Connection, prompt } = require("./utils");
+
+exports.default = async function(argv) {
+  let sqlQuery = await prompt("Enter SQL (e.g. `SELECT * FROM nodes`):");
+
+  let connection = new Connection(argv.endpoint);
+  let session;
+  try {
+    session = await connection.open();
+  } catch (error) {
+    console.error("Connection failure due to:", error);
+    return;
+  }
+
+  let result;
+  try {
+    result = await session.call(
+      `com.nearprotocol.${argv["chain-id"]}.explorer.select`,
+      [sqlQuery]
+    );
+  } catch (error) {
+    console.error("Failed to call the query function due to:", error);
+    return;
+  } finally {
+    connection.close();
+  }
+
+  console.log("Query result:");
+  console.log(result);
```

Comment:
at a later time this could be very useful to import `fs` and have the option to pipe or write to a file. Currently, you can technically run:
`./bin/near-explorer --endpoint http://localhost:8080/ws --chain-id localhostnet query > test.txt`
Not see the query prompt, but enter the query and save the results along with the prompt to a file. That's good enough for now. Talking stretch goals here.